src/components/DialogBody.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import CloseIcon from './CloseIcon';
+
+const DialogBody = ({ locationDetails, onClick }) => (
+  <>
+    <button
+      className="dialog__button"
+      aria-label="close dialog"
+      onClick={onClick}
+    >
+      <CloseIcon />
+    </button>
+
+    <div className="dialog__image">
+      <img
+        src={
+          locationDetails?.thumbnail?.source ||
+          `${process.env.PUBLIC_URL}/placeholder.png`
+        }
+        alt={locationDetails?.thumbnail?.source ? locationDetails.title : ''}
+      />
+    </div>
+    <div className="dialog__body">
+      <a
+        className="dialog__body__link"
+        href={`https://en.wikipedia.org/?curid=${locationDetails.pageid}`}
```

Comment:
It's only one suggestion **(no pressure to do it 😄).**

Last PR, Lauren suggested to us that we could use `aria-label` to links.

Reply:
good idea, we'll make sure to add those, thank you @alxmcr 😄